eng/common/scripts/Service-Level-Readme-Automation-Functions.ps1

```diff
@@ -0,0 +1,90 @@
+
+function create-service-readme($readmeFolder, $readmeName, $moniker, $msService, $indexTableLink, $serviceName, $author, $msAuthor)
+{
+  $readmePath = Join-Path $readmeFolder -ChildPath $readmeName
```

Comment:
This functions are difficult to review given they were moved. Did you change anything in them?